dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/DefaultIdentifiableObjectManager.java

```diff
@@ -1036,6 +1027,16 @@ public <T extends IdentifiableObject> List<T> getAllNoAcl(@Nonnull Class<T> type
     return store.getAllNoAcl();
   }
 
+  @Override
+  public void persist(Object userAdmin) {
+    entityManager.persist(userAdmin);
```

Comment:
with no context, this var name seems odd. Is this an object that being persisted? Why is it called userAdmin?